README.md

```diff
@@ -60,36 +60,32 @@ After you're finished please remove all the comments and instructions!
 
 <!-- PROJECT DESCRIPTION -->
 
-# 📖 [your_project_name] <a name="about-project"></a>
+# 📖 [School Library] <a name="about-project"></a>
 
 > Describe your project in 1 or 2 sentences.
 
-**[your_project__name]** is a...
+**[school library]** is a OOP and Ruby based application to ecord what books are in the library and who borrows them.
```

Comment:
- [ ] [optional] It would have been nice if you corrected spelling error in the name record

Reply:
- [ ] kindly remove the commented section of the readme file, its not good practice to have commented code in the codebase